examples/arrays/functions.md

```diff
@@ -1,38 +1,46 @@
 # Array Functions
 
-## `repeat` 
+## `repeat`
 
 Syntax
 
 ```go
-Type[element].repeat(number int)
+array.repeat(number int)
 ```
 
 Makes an array with the given element number of times.
 
 ```go
-foo := int[1].repeat(7)
+foo := [1, 2].repeat(5)
 println(foo)
 ```
+
 Output
 
 ```go
-[1, 1, 1, 1, 1, 1, 1]
+[1, 2, 1, 2, 1, 2, 1, 2, 1, 2]
 ```
 
-## insert
+## `insert`
 
 Syntax
 
 ```go
 array.insert(num int, data T)
 ```
 
-Inserts the data at a given position and shifts the elements in the array
+Inserts the data at a given position and shifts the elements in the array.
+
+<!--

```

Comment:
Why HTML Comment used there?

Reply:
It was probably his editor + its shortcuts lol.

V comment style is preferred. Please check the instructions [here](https://github.com/v-community/v_by_example/blob/master/examples/hello_world/comment.md).

Reply:
@Delta456 that markdown comment too

Reply:
@Delta456 @vbrazo It a workaround code... the original code is not work

Reply:
If it's broken, I'd prefer to remove the code and add a `# TODO:` instead, so other contributors could come and complete the example.